wwwroot/phpbb/styles/prosilver/template/overall_header.html

```diff
@@ -51,6 +51,9 @@
 
 <!-- EVENT overall_header_stylesheets_after -->
 
+<!-- Pull in FontAwesome because using images for icons is baaaad -->
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```

Comment:
The link is supposed to be like that. Why no external source? Implementing directly is significantly more difficult...